src/main/java/bio/terra/service/snapshotbuilder/query/TableVariable.java

```diff
@@ -1,15 +1,16 @@
 package bio.terra.service.snapshotbuilder.query;
 
+import bio.terra.common.exception.InternalServerErrorException;
 import jakarta.annotation.Nullable;
 import org.stringtemplate.v4.ST;
 
-public final class TableVariable implements SqlExpression {
+public class TableVariable implements SqlExpression {
```

Comment:
TableVariable cannot be final because other classes like `Concept` inherit from it